deploy/cloudformation/elastic-agent-ec2-cnvm.yml

```diff
@@ -167,14 +167,25 @@ Resources:
       UserData: !Base64
         Fn::Sub: |
           #!/bin/bash -x
+          signal() {
+            sudo yum install -y aws-cfn-bootstrap || { sudo apt-get update && sudo apt-get -y install python3-pip && sudo pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz; }
+            cfn-signal --exit-code "$1" --stack ${AWS::StackName} --resource ElasticAgentEc2Instance --region ${AWS::Region}
+          }
+          is_version_valid() {
+            local min_version="$(echo "$2" | sed --regexp-extended 's/^[a-zA-Z-]+-(.*)-[[:digit:]]{4}(-[[:digit:]]{2}){5}.yml/\1/g')"
+            [[ "$min_version" = CFT_* ]] && return 0 # if the file is not published do not check
+            printf '%s\n%s\n' "$min_version" "$1" | sort --version-sort --check=quiet
+          }
+          if ! is_version_valid "${ElasticAgentVersion}" 'CFT_VERSION' &> /dev/null; then
```

Comment:
The `CFT_VERSION` placeholder is being replaced with something in the form of `cloudformation-cnvm-8.11.0.yml`, which I don't think is what you expect.
We have two options:
1. Add another parameter, `ELASTIC_VERSION` to be replaced with `8.11.0`.
2. Replace `CFT_VERSION` with `8.11.0` without the extra data that I see there and put that data hard coded on the template.

Hope the two options are clear, I would go with 2 but it's up to you.

Reply:
Thanks for the feedback. 
I went with the 2nd option, `CFT_VERSION` is being rendered having only the version string, and the remote (s3) file name is being created inside `scripts/publish_cft.sh` and also in cloudformation files while setting the tag:

e.g.
```
--tag=cft_version:cloudformation-cspm-organization-account-CFT_VERSION.yml
```